stories/topics/proc-aap-aws-modify-cidr-ranges.adoc

```diff
@@ -1,42 +1,57 @@
 [id="proc-aap-aws-modify-cidr-ranges"]
 
-= Modify Deployment CIDR Range
+= Modifying the VPC CIDR range after deployment
```

Comment:
Instead of `VPC` maybe we can use `Application`, since we will modify the CIDR range in not only the VPC, but also subnets.

Reply:
Also, the title here is `Modifying the VPC CIDR range after deployment`, but updating CIDR after deployment is not supported, and requires the stack to be deleted and redeployed to update the CIDR.

Perhaps we can call this `Modifying the CIDR range before deployment` and draw attention to the note that says it cant be modified after.